### PR TITLE
Unpin pandas

### LINKED ISSUE #69: Remove upper bound for pandas
### Problem Description

With the newly released [pandas 2.0](https://pandas.pydata.org/docs/dev/whatsnew/v2.0.0.html) it would nice to support it across the sdv-dev libraries.
 
### Expected behavior

The newly expected version for pandas should be only `>=` than the minimum supported without upper bound. After the changes are done, both minimum and newest version should work properly.

### PR DESCRIPTION
**Pull request by frances-h**
Resolve #69

### CODE REVIEW

**Review by fealho — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
 install_requires = [
     "numpy>=1.20.0,<2;python_version<'3.10'",
-    "numpy>=1.23.3,<2;python_version>='3.10'",
-    "pandas>=1.1.3,<2;python_version<'3.10'",
+    "numpy>=1.23.3;python_version>='3.10'",
```
Comment (pvk-developer) [inline setup.py:16]: I think that this has to stay as it is

File: setup.py [hunk 2]
```diff
@@ -13,8 +13,8 @@
...
+    "pandas>=1.1.3;python_version<'3.10'",
     "pandas>=1.3.4,<2;python_version>='3.10'",
```
Comment (pvk-developer) [inline setup.py:18]: I think this one has to be changed.

**Review by pvk-developer — CHANGES_REQUESTED** [review]
I think you edited the wrong line on the setup.py, after that it should be good to merge ;)

File: setup.py (hunk 1 above)
Comment (frances-h) [reply setup.py]: good catch, thanks!

**Review by pvk-developer — APPROVED** [review]
:shipit:

**Pull request merged**